joomla/joomla-cms
# [4.0] Namespace com_finder

Pull Request for Issue # .

### Summary of Changes
As title says, this PR converts com_finder to namespace MVC for Joomla 4.

### Testing Instructions
1. Install latest Joomla 4.0-dev branch
2. Test and make sure com_finder works as expected.

### Known issues (also happens before this PR, so can be fixed on different PR)
- On frontend, click on Advanced Search, nothing happens
- RSS feed has some notices/warning.
